code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -388,9 +388,9 @@
 		msg += "<span class = 'deptradio'>Physical status:</span> <a href='?src=[UID()];medical=1'>\[[medical]\]</a>\n"
 		msg += "<span class = 'deptradio'>Medical records:</span> <a href='?src=[UID()];medrecord=`'>\[View\]</a> <a href='?src=[UID()];medrecordadd=`'>\[Add comment\]</a>\n"
 
-
-	if(print_flavor_text() && !skipface)
-		msg += "[print_flavor_text()]\n"
+	if(print_flavor_text() && get_organ("head"))
```

Comment:
I'd put `!skipface` in here instead of get_organ("head"). Then retrieve the head organ if this if statement is true retrieve the head and cache it in a variable.
The linter is also failing due to you directly using the output of `get_organ` to check `.disfigured`.